config/openhpc.yml

```diff
@@ -165,4 +165,5 @@ openhpc_packages:
   - readline-devel
   - python-devel
   - git-lfs
+  - perf
```

Comment:
This is for Peter right?

Reply:
Yep!

Reply:
And he was requesting `perl-devel` as well?

Reply:
Good point!